### ISSUE TITLE
Camera Calibrations

### ISSUE THREAD

**Issue by jsaunders909** [uniBruce/Mead#22]
Hi, congrats on the fantastic dataset! It would be really helpful if it were possible to extract 3D data from these videos. Do you have the camera calibrations for the cameras you used?

**Comment by radekd91** [comment]
Agreed both camera intrinsics and extrinsics would be extremely helpful and would make this dataset a lot better.

**Comment by HowieMa** [comment]
Same question. If the authors can provide camera information, that would be very helpful.
